src/test/scala/io/dustinsmith/bitinterleave/BinarySpec.scala

```diff
@@ -15,36 +15,31 @@
  */
 package io.dustinsmith.bitinterleave
 
-import java.io.File
-
-import Binary.getBinaryFunc
-import org.scalatest.BeforeAndAfterAll
-import org.scalatest.PrivateMethodTester
+import io.dustinsmith.SparkSessionTestWrapper
```

Comment:
Import order style

```xml
    <check level="error" class="org.scalastyle.scalariform.ImportOrderChecker" enabled="true">
        <parameters>
            <parameter name="groups">java,scala,3rdParty,spark</parameter>
            <parameter name="group.java">javax?\..*</parameter>
            <parameter name="group.scala">scala\..*</parameter>
            <parameter name="group.3rdParty">(?!org\.apache\.spark\.).*</parameter>
            <parameter name="group.spark">org\.apache\.spark\..*</parameter>
        </parameters>
    </check>
```

java on top, scala, 3rd party, spark

Reply:
@dwsmith1983  Could you please advise how to apply formatting according to the `scalastyle-config.xml`? I'm usually using `.scalafmt.conf` for that...

Reply:
@subpath I added git hooks in the README branch. I see I never updated and had some updates from 2021. If you merge this branch over, you can just add the git hook to process the scalastyle for you
https://github.com/dwsmith1983/space-filling-curves/tree/feature/update-readme-doc

Reply:
Need to fix tests after pulling from the [feature/update-readme-doc](https://github.com/dwsmith1983/space-filling-curves/tree/feature/update-readme-doc) branch... I will do it a bit latter